src/client/components/app/Transactions/ReleaseCollateralTx.tsx

```diff
@@ -247,7 +249,7 @@ export const ReleaseCollateralTx: FC<ReleaseCollateralTxProps> = (props) => {
       <StyledTransaction onClose={onClose} header={'transaction'}>
         <TxStatus
           success={transactionCompleted}
-          transactionCompletedLabel={'could not add credit'}
+          transactionCompletedLabel={'could not release collateral'}
```

Comment:
translation service plz

Reply:
lots of different instances of this for txns associated with modals. 

moved all of them into translation service